solutions/standard/DA-types.md

```diff
@@ -0,0 +1,51 @@
+# Configuring complex inputs for KMS in IBM Cloud projects
+
+Several optional input variables in the IBM Cloud [KMS all inclusive deployable architecture](https://cloud.ibm.com/catalog#deployable_architecture) use complex object types. You specify these inputs when you configure deployable architecture.
+
+* Rules For Context-Based Restrictions (`cbr_rules`)
```

Comment:
Rules are for the service (KMS in this case) which is known as CBR rules, you need to update this accordingly.

Reply:
fixed https://github.com/terraform-ibm-modules/terraform-ibm-kms-all-inclusive/pull/594/commits/806305d97382a08396e3ba9111e5efdbda2cf2cc